lib/tasks/migrations/migrations.rake

```diff
@@ -39,10 +40,14 @@ namespace :migrations do
     start, stop = range(args)
 
     has_encountred_issue = 0
+    filtered_roles_names = Role::RESERVED_ROLE_NAMES - %w[admin user]
 
     User.select(:id, :uid, :name, :email, :social_uid, :language, :role_id)
+        .joins(:role)
+        .where.not(roles: { name: filtered_roles_names }, deleted: true)
```

Comment:
No deleted, pending, denied or super_admin users will be migrated,

Reply:
Do we need to explicitly check `deleted: true` ? Isn't that the default scope?

Reply:
Yup, I did it to be independent of the original version of Greenlight v2 and being more specific to be compatible with the largest number possible of deployment (most of other forked version),
Isn't that desired?